src/main/resources/application.properties

```diff
@@ -1,5 +1,5 @@
 spring.application.name=configserver
 spring.profiles.active=native
-spring.cloud.config.server.native.searchLocations=${HOME}/ngs_3/config
-spring.cloud.config.server.git.uri=${HOME}/ngs_3/config
-server.port=8888
\ No newline at end of file
+spring.cloud.config.server.native.searchLocations=/etc/ega-data-api/
```

Comment:
Unfortunately, I don't have permissions to create a directory in /etc on out production servers.